### ISSUE TITLE
NTR: Cationic trypsinogen low in blood (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8207]
New term request Cationic trypsinogen low in blood (segal_080822211819):

**Comment by pnrobinson** [comment]
@MickeySegal
is cationic trypsinogen distinct from 
[Increased circulating trypsinogen HP:0032493](https://hpo.jax.org/app/browse/term/HP:0032493)
what is the context?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1756/
"Assessment of serum concentration of the digestive enzyme cationic trypsinogen"

https://www.sciencedirect.com/topics/neuroscience/trypsinogen lists 
"In vitro autoactivation and autolysis properties of the three human trypsin(ogen)s [26,38,41,42] are compared in Table 576.3. As indicated in the table, mesotrypsinogen is deficient in autoactivation and is significantly impaired in cleavage of anionic and cationic trypsinogens at the autolysis site; the deficiency in autoactivation is rescued by mutation of Arg198 to Gly [26]."

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000968
- label: Diminished circulating cationic trypsinogen concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:3792918
- def: Concentration of cationic trypsinogen in the blood circulation below the lower limit of normal.
- comment: Cationic trypsinogen is one of the several isoforms of trypsinogen. Serum cationic trypsinogen when low, is known to be associated with reduced pancreatic exocrine function in a number of disorders, including Shwachman's syndrome and type 1 diabetes mellitus.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8207
- synonyms:
